lua/neo-tree/sources/common/preview.lua

```diff
@@ -270,12 +270,30 @@ function Preview:activate()
   vim.api.nvim_win_set_var(self.winid, "neo_tree_preview", 1)
 end
 
+---@param winid number
+---@param bufnr number
+local function load_image_nvim_buf(winid, bufnr)
```

Comment:
I think this should be called `try_load_image_nvim_buf` or `load_image_nvim_buf_if_image` or something similar.

Reply:
Changed to `try_load_image_nvim_buf`